runner/update-status

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+if [[ ${1:-} == '' ]]; then
+  # shellcheck source=runner/logger.bash
+  source logger.bash
+  log.error "Missing required argument -- '<phase>'"
+  exit 64
+fi
+
+if [ "${RUNNER_STATUS_UPDATE_HOOK:-false}" == "true" ]; then
+
```

Comment:
We should never use `test` (`[]`) because it works weirdly compared Bash’s `[[]]`.

```suggestion
if [[ ${RUNNER_STATUS_UPDATE_HOOK:-false} == true ]]; then
```